scripts/py_matter_yamltests/matter_yamltests/websocket_runner.py

```diff
@@ -93,15 +100,48 @@ async def _stop_client(self, instance):
         if instance:
             await instance.close()
 
-    async def _start_server(self, command):
+    async def _start_server(self, command, url):
         instance = None
         if command:
-            instance = subprocess.Popen(command, stdout=subprocess.DEVNULL)
+            start_time = time.time()
+
+            command = ['stdbuf', '-oL', '-eL'] + command  # disable buffering
+            instance = subprocess.Popen(
+                command, text=True, bufsize=1, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+
+            # Loop to read the subprocess output with a timeout
+            lines = []
+            while True:
+                if time.time() - start_time > _WEBSOCKET_SERVER_MESSAGE_TIMEOUT:
+                    for line in lines:
+                        print(line, end='')
+                    self._hooks.abort(url)
+                    instance.kill()
```

Comment:
Should this also do the "terminate, then kill" thing?  Might want that in a helper method....

Reply:
I used `self._stop_server`